src/services/completions.ts

```diff
@@ -335,9 +340,23 @@ namespace ts.Completions {
                 continue;
             }
 
+            let sortText = symbolToSortTextMap && symbolToSortTextMap[getSymbolId(symbol)];

```

Comment:
I don't think you want this change. SortText should be set when adding the member instead